templates/post.ftl

```diff
@@ -21,57 +18,20 @@
         </div>
         <hr style="margin-bottom: 5px;"/>
         <div style="margin-bottom: 20px;">
-            <div class="pull-right">
-                <a href="${config.canonicalBaseUrl}/${content.uri}"><span class="label label-default">Permalink</span></a>
+            <div class="float-end">
+                <a href="${config.canonicalBaseUrl}/${content.uri}"><span class="badge bg-dark">Permalink</span></a>
             </div>
             <div class="tags">
-                <span class="glyphicon glyphicon-tags"></span>
-                &nbsp;tagged as
+                <i class="fas fa-tag"></i>&nbsp;tagged as
                 <#list content.tags as tag>
-                    <a href="${content.rootpath}blog/tags/${tag?url}.html"><span class="label label-info">${tag}</span></a>
+                    <a href="${content.rootpath}blog/tags/${tag?url}.html"><span class="badge bg-info">${tag}</span></a>
                 </#list>
             </div>
         </div>
     </div>
     <div class="comments">
         <h2>Comments</h2>
-        <a class="btn btn-default" href="${config.googleGroupURL}" style="margin-left: 20px">Visit our forum to comment</a>
+        <a class="btn btn-secondary" href="${config.googleGroupURL}" style="margin-left: 20px">Visit our forum to comment</a>
```

Comment:
We should probably drop the inline styles and use the utility classes from bootstrap

Reply:
Done, using mb-4 etc in all templates.